admin_guide/diagnostics_tool.adoc

```diff
@@ -115,3 +115,54 @@ A client with *cluster-admin* access available (for any user, but only the
 current master) should be able to diagnose the status of infrastructure such as
 nodes, registry, and router. In each case, running `oc adm diagnostics` looks
 for the client configuration in its standard location and uses it if available.
+
+[[additional-cluster-health-checks]]
+== Additional Diagnostic Checks via Ansible
+
+Some additional diagnostic checks are available through the https://hub.docker.com/r/openshift/openshift-ansible[openshift-ansible]
+Docker image.
```

Comment:
and, it looks like the name will be `origin-openshift-ansible` in the near future (and `ose-` for the OCP version).